sudokuProject/src/sudoku/Settings.java

```diff
@@ -1,5 +1,69 @@
 package sudoku;
 
+import org.w3c.dom.DOMException;
+
+import sudoku.io.XMLHandler;
+
 public class Settings {
+	private String		level;
```

Comment:
do not use this kind of alignment it is better keep the attributes using normal indentation without align them